tutorials/hist/Normalizing_Histogram.C

```diff
@@ -0,0 +1,171 @@
+/// \file
+/// \ingroup tutorial_hist
+/// \notebook -js
+/// Normalizing a Histogram
+///
+/// Image produced by `.x Normalizing_Histogram.C`
+/// 7 different methods of normalizing Histograms
+/// are shown, each with the original Histogram
+/// next to the normalized one.
+/// \macro_image
+/// \macro_code
+///
+/// The methods are:
+/// 1:
+/// ```
+/// Double_t num = h->GetBinContent(i);
+/// Double_t den = h->GetBinWidth(i);
+/// Double_t value = 0;
+/// if (den!=0)
```

Comment:
The bin width cannot really be 0 - the tutorial shouldn't bother to check, and especially not the doc.

Reply:
Okay. I removed that method anyway so it shouldn't matter. The only methods are the one with "width" and the one without, so a total of 2.